occurrence-hadoop-minicluster/pom.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
shouldn't this just inherit some dependencies or versions from the parent pom?

Reply:
I've reverted this, and i've removed the reference to the parent pom.

This module is a bit of dependency nightmare with hbase and hadoop, and combined with the `occurrence-integration-tests` module has caused quite a few headaches.

It all works fine if `occurrence-hadoop-minicluster` is built as a separate module, and in a separate maven build run (see Jenkins file for the separate pipelines stage for building this submodule).

As this test code, Im inclined to live with it as is. Optionally the `occurrence-hadoop-minicluster` could be moved out of the `occurrence` repository, which is perhaps a cleaner option (there isnt anything "occurrence" specific about this module).